squad-server/plugins/db-log.js

```diff
@@ -503,6 +509,7 @@ export default class DBLog extends BasePlugin {
       attackerSquadID: info.attacker ? info.attacker.squadID : null,
       damage: info.damage,
       weapon: info.weapon,
+      role: info.role,
```

Comment:
How are you getting the role here if it's only in the die event log line?

Not sure I'm a fan of storing this information given it's very poor, i.e. only present on the die event. Probably better to get it another way, e.g. via Possess log lines, or wait until OWI add something.

Reply:
You are right... It only works for Deaths and not Wounds 👍 
For now we may just log the role in Deaths and wait for OWI

I didn't realise that at first place thought it would just grab it from the Deaths, my bad for not testing it at first place. Sorry 😞

Reply:
Have you looked at what the possess log line provides? I think the info might be stored in `server.players` too.